test/helpers/web3/getWeb3.js

```diff
@@ -0,0 +1,21 @@
+const Ganache = require(process.env.TEST_BUILD
+  ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
+  : "../../../index.js");
+const Web3 = require("web3");
+
+const getWeb3 = async(options = {}) => {
+  const provider = Ganache.provider(options);
+  const web3 = new Web3(provider);
+  const accounts = await web3.eth.getAccounts();
+
+  return Object.assign(
```

Comment:
Why `Object.assign` here?

Reply:
Can change to {}.

Reply:
Changes were made to `initializeTestProvider.js`.